run-development.sh

```diff
@@ -19,9 +19,13 @@ cd /usr/src/app/
 # Install dependencies
 ######################
 
-## Check if package.json existed and did not change since previous build (/usr/src/app/app/ is copied later in this script, at first run from the template itself it doesn't exist but that's fine for comparison)
-cmp -s /app/package.json /usr/src/app/app/package.json
+## Check if package.json existed and did not change since previous build (at first run from the template itself it doesn't exist but that's fine for comparison)
+cmp -s /app/package.json /usr/src/service-package-old.json
```

Comment:
need to saveguard the original now as we will merge service and template together which results in a change of this file